<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glView</class>
 <widget class="QWidget" name="glView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>glView</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>781</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Shift</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="7" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>241</red>
            <green>240</green>
            <blue>250</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>8</red>
            <green>8</green>
            <blue>9</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>208</red>
            <green>208</green>
            <blue>208</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>64</red>
            <green>65</green>
            <blue>66</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="96">
            <red>164</red>
            <green>166</green>
            <blue>168</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>8</red>
            <green>8</green>
            <blue>9</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDoubleSpinBox" name="scaleX">
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QDoubleSpinBox" name="rotateZ">
      <property name="maximum">
       <double>359.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QDoubleSpinBox" name="shiftY">
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="8">
     <widget class="QLineEdit" name="lineEditFileName"/>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>z:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>x:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>x:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QDoubleSpinBox" name="rotateY">
      <property name="maximum">
       <double>359.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QLabel" name="v_count">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButtonFile">
      <property name="text">
       <string>Choose file</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QDoubleSpinBox" name="shiftZ">
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>z:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QDoubleSpinBox" name="scaleZ">
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>x:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QDoubleSpinBox" name="rotateX">
      <property name="maximum">
       <double>359.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QDoubleSpinBox" name="scaleY">
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QLabel" name="p_count">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="v_label">
      <property name="text">
       <string>Vertexes:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLabel" name="p_label">
      <property name="text">
       <string>Polygons:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>z:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>y:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Rotate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QDoubleSpinBox" name="shiftX">
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>y:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>y:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Scale</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="QSpinBox" name="bc1"/>
    </item>
    <item row="0" column="9">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Background:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QSpinBox" name="bc2"/>
    </item>
    <item row="0" column="11">
     <widget class="QSpinBox" name="bc3"/>
    </item>
    <item row="1" column="9">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Vertex:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QSpinBox" name="vc"/>
    </item>
    <item row="1" column="11">
     <widget class="QSpinBox" name="vc2">
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="12">
     <widget class="QSpinBox" name="vc3"/>
    </item>
    <item row="2" column="9">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Polygon:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="10">
     <widget class="QSpinBox" name="pc1"/>
    </item>
    <item row="2" column="11">
     <widget class="QSpinBox" name="pc2"/>
    </item>
    <item row="2" column="12">
     <widget class="QSpinBox" name="pc3">
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="3" column="9">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Vertex size:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="10">
     <widget class="QSpinBox" name="vsize">
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="3" column="11" colspan="2">
     <widget class="QCheckBox" name="quads">
      <property name="text">
       <string>Quads</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
